keep/providers/providers_factory.py

```diff
@@ -48,7 +48,14 @@ def get_provider(
                 + provider_type_split[1].title().replace("_", "")
                 + "Provider",
             )
-        return provider_class(provider_id=provider_id, config=provider_config)
+        try:
+            return provider_class(provider_id=provider_id, config=provider_config)
+        except TypeError as exc:
+            error_message = f"Configuration problem while trying to initialize the provider {provider_id}. Probably missing provider config, please check the provider configuration. [supplied configuration: {provider_config}]]"
```

Comment:
This can also be problematic in a "hosted" version since it discloses secrets in logs

Reply:
would remove/simplify

Reply:
removed